recipe/meta.yaml

```
@@ -9,14 +9,15 @@ source:
   # we have to use git urls here to get the submodules needed for the
   # build
   git_url: https://github.com/dmlc/xgboost
-  git_tag: v1.3.0
+  git_tag: 000292ce6d99ed658f6f9aebabc6e9b330696e7e
```

Please keep the named tag
```suggestion
  git_tag: v1.3.3
```